beacon-chain/state/state-native/getters_attestation.go

```diff
@@ -19,11 +26,19 @@ func (b *BeaconState) PreviousEpochAttestations() ([]*ethpb.PendingAttestation,
 // previousEpochAttestationsVal corresponding to blocks on the beacon chain.
 // This assumes that a lock is already held on BeaconState.
 func (b *BeaconState) previousEpochAttestationsVal() []*ethpb.PendingAttestation {
```

Comment:
What does Val stand for ? Why does it return nil instead of err? I guess it's unrelated to this PR, but would be useful to understand why we have that pattern

Reply:
We always had this pattern in the state, although previously exported and imported function names differed only by casing. To make them stand out more, we added the `Val` suffix (denoting `value`).